distributed/protocol/tests/test_cupy.py

```diff
@@ -29,3 +29,23 @@ def test_serialize_cupy_from_numba(dtype):
     y = deserialize(header, frames, deserializers=("cuda", "dask", "pickle", "error"))
 
     assert (x_np == cupy.asnumpy(y)).all()
+
+
+@pytest.mark.parametrize("size", [0, 3, 10])
+def test_serialize_cupy_from_rmm(size):
```

Comment:
Not sure I understand this test, why is the original array a NumPy one instead of a CuPy array?

Reply:
Nevermind, I just saw other tests are written in the same way, so no reason to change things.

Reply:
No worries. Yeah it seemed prudent to test building other objects from RMM `DeviceBuffer`'s more directly since this is something we implicitly rely on for all GPU objects (when RMM is available).